test_pull_requests

```diff
@@ -131,12 +131,45 @@ end
 GITHUB_API_BASE_URL = 'https://api.github.com'
 GITHUB_BASE_URL = 'https://github.com'
 
+CONTENT_WAITING_IN_QUEUE = "Waiting: You are in the build queue at position:"
+CONTENT_WAITING_DETERMINING_QUEUE_POS = "Waiting: Determining build queue position"
+CONTENT_WAITING_STABLE_BUILD = "Waiting for stable build of"
+CONTENT_RUNNING = "Running"
+CONTENT_EVALUATING = "Evaluating for testing"
+
+CONTENT_TO_STATE = {
+  CONTENT_EVALUATING                    => :evaluating,
+  CONTENT_RUNNING                       => :running,
+  CONTENT_WAITING_IN_QUEUE              => :wait_in_queue,
+  CONTENT_WAITING_DETERMINING_QUEUE_POS => :wait_queue_pos,
+  CONTENT_WAITING_STABLE_BUILD          => :wait_stable_build,
+  "SUCCESS"                             => :success,
+  "FAILURE"                             => :failure,
+  "ABORTED"                             => :aborted,
+  "UNSTABLE"                            => :unstable,
+  "NOT_BUILT"                           => :not_built,
+  "NOTFOUND"                            => :not_found
```

Comment:
Change this to NOT_FOUND to be consistent and use a constant throughout the file.